src/endpoints/rasters.js

```diff
@@ -1,25 +1,21 @@
-import { request } from '../http';
-import RasterStore from '../valueobjects/RasterStore';
-import RasterStoreDetail from '../valueobjects/RasterStoreDetail';
-import Organisation from '../valueobjects/Organisation';
+import { endpoint, request } from '../http';
+import { processSingleResultResponse, processMultipleResultsResponse } from '../tools';
 
 // Retrieve rasters from the API rasterstore list page.
 export function getRasters(filters = {}) {
-  return request('/rasters/', filters).then(function (results) {
-    return results.map(function (result) {
-      if (result.organisation) {
-        result = result.set('organisation', new Organisation(result.organisation));
-      }
-      return new RasterStore(result);
-    });
+  let url = endpoint('/rasters/', filters);
+
+  console.log('URL: ', url);
```

Comment:
Laten we maar zo min mogelijk console.logs naar master mergen; dan gaan we binnenkort eens een mooi generiek log systeempje bekokstoven.